src/99-appendices/11-qmri.md

```diff
@@ -0,0 +1,558 @@
+    by defining a set of logical conditions over the metadata fields, the tables of the
+    [deriving the intended qMRI application from an ambiguous file collection](#Deriving-the-intended-qMRI-application-from-an-ambiguous-file-collection)
+    and the
+    [anatomy data priority levels](#Anatomy-imaging-data)
+    sections are extended instead of introducing a new suffix.
+
+## Application-specific notes for qMRI file collections
+
+### Anatomy imaging data
+
+General notes:
+
+-   Some BIDS metadata field values are calculated based on the values of other metadata fields that are not listed as required fields.
+    These fields include: `NumberShots`.
+    The calculation of the values may depend on the type of the acquisition.
+    These acquisitions include: `MP2RAGE` and `TB1SRGE`.
+
+#### `MP2RAGE` specific notes
+
+##### `UNIT1` images
+
+Although the `UNIT1` image is provided as an output by the acquisition sequence, it is used
+as an input to offline calculation of a `T1map` using a dictionary lookup approach. However,
+`complex` data is needed for an accurate calculation of the `UNIT1` image, which is not commonly
+provided by the stock sequence. Instead, the `magnitude` and `phase` images are exported. Please
+see the relevant discussion [here](https://github.com/qMRLab/qMRLab/issues/255).
+
+Therefore, the `UNIT1` image provided by the scanner is RECOMMENDED to be stored under the `anat`
+raw dataset directory along with the `MP2RAGE` file collection and to be used as the primary input
+for quantifying a `T1map`.
+
+If an additional `UNIT1` image is calculated offline, then the output is to be stored in the
+`derivatives` folder with neccesary provenance information.
+
+##### `NumberShots` metadata field
+
+Note that type of `NumberShots` field can be either a `number` or an `array of numbers`.
+
+-   If a single `number` is to be provided, this should correspond to the number of `SlicesPerSlab` or `ReconMatrixPE`.
+    However, in this case, `SlicePartialFourier` or `PartialFourierPE` fraction is needed
+    to calculate the number of partitions `before` and `after` of the k-space center to calculate a T1 map.
+
+-   If `before/after` calculation is performed during the BIDS conversion of the `MP2RAGE` data,
+    then the value of `NumberShots` metadata field can be given as a 1X2 array,
+    with first entry corresponding to `before` and the second to the `after`.
+
+Formula:
+
+If NumberShots is an array of numbers such that `"NumberShots": [before, after]`,
+the values of `before` and `after` are calculated as follows:
+
+```text
+before = SlicesPerSlab*(SlicePartialFourier - 0.5)
+after  = SlicesPerSlab/2
+```
+
+See this [reference](https://github.com/JosePMarques/MP2RAGE-related-scripts/blob/a405df30ac2c617d29d8b1b16025aaa911e86370/func/bids_T1B1correct.m#L16).
+
+##### Other metadata fields
+
+The value of the `RepetitionTimeExcitation` is not commonly found in the DICOM files.
```

Comment:
"value of the `RepetitionTimeExcitation`" --> "value of the `RepetitionTimeExcitation` field"

Reply:
3f33e1d